src/features/dataset-controls/DatasetControls.module.css

```diff
@@ -88,7 +88,9 @@
   display: flex;
   flex-direction: row no-wrap;
   align-items: flex-start;
-  white-space: nowrap;
```

Comment:
revert these please. I don't want that block to wrap because it throws off the spacing of the control pane. The text extending on hover was intentional.